lib/db/mysql.js

```diff
@@ -777,6 +777,13 @@ module.exports = function (log, error) {
     )
   }
 
+  // Get : email
+  // Values : email = $1
+  var GET_EMAIL = 'CALL getEmail_1(?)'
+  MySql.prototype.getEmail = function (emailBuffer) {
```

Comment:
@vladikoff Hrm, yea that is confusing. I can think of a couple possible solutions for this. 

* Extend `getEmail` to return an email from the accounts table or emails table and make the response be consistent with an email object (`isVerified`, `isPrimary`, etc).
* Rename to `getSecondaryEmail`. This doesn't quite follow the pattern for other emails endpoints but gives a clearer understanding of what it does.

I am leaning towards option 2 because that is a less intensive db call and this check would happen often. Thoughts?